sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_do_while.py

```diff
@@ -4,7 +4,7 @@
 from azure.ai.ml.entities._builders.do_while import DoWhile
 
 
-def do_while(body, condition, mapping, *, max_iteration_count=None):
+def do_while(body, mapping, max_iteration_count: int, condition=None):
```

Comment:
BTW why mapping is a required param?

Reply:
@D-W- any comment on this?
I understand it can be optional, for those loop body without input?

Reply:
Opened a task tracking empty mapping and optional.